packages/packages.csv

```diff
@@ -1,174 +1,174 @@
+Microsoft.AspNetCore.MiddlewareAnalysis,ship,include
+Microsoft.AspNetCore.Mvc,ship,include
+Microsoft.AspNetCore.Mvc.Abstractions,ship,include
+Microsoft.AspNetCore.Mvc.ApiExplorer,ship,include
+Microsoft.AspNetCore.Mvc.Core,ship,include
+Microsoft.AspNetCore.Mvc.Cors,ship,include
+Microsoft.AspNetCore.Mvc.DataAnnotations,ship,include
+Microsoft.AspNetCore.Mvc.Formatters.Json,ship,include
+Microsoft.AspNetCore.Mvc.Formatters.Xml,ship,include
+Microsoft.AspNetCore.Mvc.Localization,ship,include
+Microsoft.AspNetCore.Mvc.Razor,ship,include
+Microsoft.AspNetCore.Mvc.Razor.Host,ship,include
+Microsoft.AspNetCore.Mvc.TagHelpers,ship,include
+Microsoft.AspNetCore.Mvc.ViewFeatures,ship,include
+Microsoft.AspNetCore.Mvc.WebApiCompatShim,ship,include
+Microsoft.AspNetCore.Owin,ship,include
+Microsoft.AspNetCore.Proxy,ship,include
+Microsoft.AspNetCore.Razor,ship,include
+Microsoft.AspNetCore.Razor.Runtime,ship,include
+Microsoft.AspNetCore.Razor.Test.Sources,noship,exclude
+Microsoft.AspNetCore.Routing,ship,include
+Microsoft.AspNetCore.Routing.Abstractions,ship,include
+Microsoft.AspNetCore.Routing.DecisionTree.Sources,noship,exclude
+Microsoft.AspNetCore.Server.IISIntegration,ship,include
+Microsoft.AspNetCore.Server.IISIntegration.Tools,ship,exclude
+Microsoft.AspNetCore.Server.Kestrel,ship,include
+Microsoft.AspNetCore.Server.Kestrel.Https,ship,include
+Microsoft.AspNetCore.Server.Testing,noship,exclude
+Microsoft.AspNetCore.Server.WebListener,ship,include
+Microsoft.AspNetCore.Session,ship,include
+Microsoft.AspNetCore.SignalR.Messaging,noship,exclude
+Microsoft.AspNetCore.SignalR.Redis,noship,exclude
+Microsoft.AspNetCore.SignalR.Server,noship,exclude
+Microsoft.AspNetCore.SignalR.ServiceBus,noship,exclude
+Microsoft.AspNetCore.SignalR.Sources,noship,exclude
+Microsoft.AspNetCore.SignalR.SqlServer,noship,exclude
+Microsoft.AspNetCore.StaticFiles,ship,include
+Microsoft.AspNetCore.TestHost,ship,exclude
+Microsoft.AspNetCore.Testing,noship,exclude
+Microsoft.AspNetCore.WebSockets.Client,noship,exclude
+Microsoft.AspNetCore.WebSockets.Protocol,ship,include
+Microsoft.AspNetCore.WebSockets.Server,ship,include
+Microsoft.AspNetCore.WebUtilities,ship,include
+Microsoft.Data.Sqlite,ship,include
+Microsoft.EntityFrameworkCore,ship,include
+Microsoft.EntityFrameworkCore.InMemory,ship,include
+Microsoft.EntityFrameworkCore.Relational,ship,include
+Microsoft.EntityFrameworkCore.Relational.Design,ship,exclude
+Microsoft.EntityFrameworkCore.Relational.Design.Specification.Tests,ship,exclude
+Microsoft.EntityFrameworkCore.Relational.Specification.Tests,ship,exclude
+Microsoft.EntityFrameworkCore.Specification.Tests,ship,exclude
+Microsoft.EntityFrameworkCore.Sqlite,ship,include
+Microsoft.EntityFrameworkCore.Sqlite.Design,ship,exclude
+Microsoft.EntityFrameworkCore.SqlServer,ship,include
+Microsoft.EntityFrameworkCore.SqlServer.Design,ship,exclude
+Microsoft.EntityFrameworkCore.Tools,ship,exclude
+Microsoft.EntityFrameworkCore.Design,ship,exclude
```

Comment:
I believe it can be excluded from the cache. This assembly is only ever used at design time, never at runtime.